.gitignore

```diff
@@ -44,3 +44,4 @@ node_modules/
 # IDE specific
 .vscode/
 yarn.lock
+package-lock.json
```

Comment:
The lock file should be committed with npm 5, this ensures that we all get the same packages. Same thing with yarn.lock